lonestar/mining/cpu/frequent-subgraph-mining/README.md

```diff
@@ -16,12 +16,12 @@ of the Meeting on Algorithm Engineering & Expermiments, 135-149.
 INPUT
 --------------------------------------------------------------------------------
 
-We support the following four input graph formats: **gr**, **txt**, **adj**, **mtx**.
+We support the following input graph formats: **txt**, **adj**.
```

Comment:
Do we not support gr? Why?

Reply:
because fsm needs labels

Reply:
Ok. Thanks. Do we support mtx for GPU but not for CPU? That's what the README for GPU says.

Reply:
mtx not supported. already removed.